environments/centos-9/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM rockylinux:9
```

Comment:
Ruby 2.7 and 3.0 don't support the OpenSSL version included in Enterprise Linux 9, so we should exclude these Ruby versions in config.yml.